PYME/recipes/output.py

```diff
@@ -357,47 +357,6 @@ def default_view(self):
                     [Item(tn) for tn in params] + [Item('_'), ] +
                     [Item(tn) for tn in outputs], buttons=['OK', 'Cancel'])
 
-#from jinja2 import Environment, PackageLoader
-import jinja2
-import os
-class UnifiedLoader(jinja2.BaseLoader):
```

Comment:
Moving this into a new module means that the existing templates under `recipes/templates` won't be found.

Reply:
Good catch

Reply:
I moved the omnipresent report templates from recipes/templates to reports/templates, which makes more sense and fixes this issue